.autorc

```diff
@@ -9,6 +9,7 @@
   "repo": "physutils",
   "name": "Stefano Moia",
   "email": "s.moia@bcbl.eu",
+  "defaultReleaseType": "none",
```

Comment:
I think this has to go into the conventional commit plugin instead?

Reply:
You might be right! Let me take a closer look